packages/lesswrong/components/messaging/FriendlyInbox.tsx

```diff
@@ -0,0 +1,275 @@
+      borderTopRightRadius: 0,
+      borderrRight: "none",
+    },
+  },
+  headerText: {
+    overflow: "hidden",
+    display: "-webkit-box",
+    "-webkit-box-orient": "vertical",
+    "-webkit-line-clamp": 1,
+    [theme.breakpoints.down('xs')]: {
+      "-webkit-line-clamp": 2,
+    }
+  },
+  actionIcon: {
+    color: theme.palette.grey[600],
+    width: 24,
+    height: 24,
+    cursor: "pointer",
+  }
+});
+
+const FriendlyInbox = ({
+  currentUser,
+  terms,
+  conversationId,
+  isModInbox = false,
+  classes,
+}: InboxComponentProps & {
+  conversationId?: string;
+}) => {
+  const { openDialog } = useDialog();
+  const { currentRoute, location } = useLocation();
+  const { history } = useNavigation();
+
+  const selectedConversationRef = useRef<HTMLDivElement>(null);
+
+  const selectConversationCallback = useCallback(
+    (conversationId: string | undefined) => {
+      history.push({ ...location, pathname: `/${isModInbox ? "moderatorInbox" : "inbox"}/${conversationId}` });
+    },
+    [history, isModInbox, location]
+  );
+
+  const openNewConversationDialog = useCallback(() => {
+    openDialog({
+      componentName: "NewConversationDialog",
+      componentProps: {
+        isModInbox,
+      },
+    });
+  }, [isModInbox, openDialog]);
+
+  const { InboxNavigation2, ConversationWidget, ForumIcon, ConversationDetails, Typography } = Components;
+
+  const conversationsResult: UseMultiResult<"ConversationsList"> = useMulti({
+    terms,
+    collectionName: "Conversations",
+    fragmentName: "ConversationsList",
+    limit: 500,
+  });
+  const { results: conversations } = conversationsResult;
+
+  // The conversationId need not appear in the sidebar (e.g. if it is a new conversation). If it does,
+  // use the conversation from the list to load the title faster, if not, fetch it directly.
+  const eagerSelectedConversation = useMemo(() => {
+    return conversations?.find((c) => c._id === conversationId);
+  }, [conversations, conversationId]);
+  const { document: fetchedSelectedConversation } = useSingle({
+    documentId: conversationId,
+    collectionName: "Conversations",
+    fragmentName: "ConversationsList",
+    skip: !conversationId,
+  });
+  const selectedConversation = fetchedSelectedConversation || eagerSelectedConversation;
+
+  const openConversationOptions = () => {
+    if (!conversationId) return;
+
+    openDialog({
+      componentName: "ConversationTitleEditForm",
+      componentProps: {
+        documentId: conversationId,
+      },
+    });
+  };
+
+  const showModeratorLink = userCanDo(currentUser, 'conversations.view.all') && !isModInbox;
+
+  const title = selectedConversation
+    ? conversationGetTitle2(selectedConversation, currentUser)
+    : "No conversation selected";
+
+  return (
+    <div className={classes.root}>
+      {showModeratorLink && (
+        <Link to={"/moderatorInbox"} className={classes.modInboxLink}>
+          Mod Inbox
+        </Link>
+      )}
```

Comment:
This is quite ugly, but it's only for admins
<img width="1208" alt="Screenshot 2023-10-26 at 14 49 46" src="https://github.com/ForumMagnum/ForumMagnum/assets/77623106/e5417131-f5dc-48c6-b27f-efe301ff5f81">